src/js/modal.js

```diff
@@ -1,156 +1,190 @@
+
+    const inputName = document.createElement("input");
+    inputName.type = "text";
+    inputName.classList.add("input-name");
     form.appendChild(inputName);
 
-    const labelEmail = document.createElement('label');
-    labelEmail.textContent = 'E-mail';
-    labelEmail.for = 'input-email';
+    const labelEmail = document.createElement("label");
+    labelEmail.textContent = "E-mail";
+    labelEmail.for = "input-email";
     form.appendChild(labelEmail);
 
-    const inputEmail = document.createElement('input');
-    inputEmail.type = 'text';
-    inputEmail.classList.add('input-email');
+    const inputEmail = document.createElement("input");
+    inputEmail.type = "email";
+    inputEmail.classList.add("input-email");
     form.appendChild(inputEmail);
 
-    const labelTime = document.createElement('label');
-    labelTime.textContent = 'What time?';
-    labelEmail.for = 'select-time';
+    const labelTime = document.createElement("label");
+    labelTime.textContent = "What time?";
+    labelEmail.for = "select-time";
     form.appendChild(labelTime);
 
-    const selectTime = document.createElement('select');
-    selectTime.classList.add('select-time');
+    const selectTime = document.createElement("select");
+    selectTime.classList.add("select-time");
+    const option = document.createElement("option");
+    option.textContent = "-- Choose time --";
+    selectTime.appendChild(option);
     form.appendChild(selectTime);
 
     for (let i = 0; i < times.slots.length; i++) {
-        const option = document.createElement('option');
+        const option = document.createElement("option");
         option.textContent = times.slots[i];
         selectTime.appendChild(option);
     }
 
-    const labelParticipants = document.createElement('label');
-    labelParticipants.textContent = 'How many participants?';
-    labelParticipants.for = 'select-participants';
+    const labelParticipants = document.createElement("label");
+    labelParticipants.textContent = "How many participants?";
+    labelParticipants.for = "select-participants";
     form.appendChild(labelParticipants);
 
-    const selectParticipants = document.createElement('select');
-    selectParticipants.classList.add('select-participants');
+    const selectParticipants = document.createElement("select");
+    selectParticipants.classList.add("select-participants");
+    const opt = document.createElement("option");
+    opt.textContent = "-- Select number --";
+    selectParticipants.appendChild(opt);
     form.appendChild(selectParticipants);
 
     let amountOfOptions = challenge.maxParticipants - challenge.minParticipants;
     for (let i = 0; i < amountOfOptions; i++) {
-        const option = document.createElement('option');
+        const option = document.createElement("option");
         option.textContent = `${challenge.minParticipants + i} participants`;
         selectParticipants.appendChild(option);
     }
```

Comment:
Denna kod kommer generera minus en deltagarna från vad som kommer ifrån API:n. 
För att lösa detta kan man göra en quick fix och bara plussa på 1.

let amountOfOptions = challenge.maxParticipants - challenge.minParticipants + 1;

Reply:
I for loopen changed from i < amountOfParticipants to <= so it solves it.